.circleci/config.yml

```diff
@@ -12,16 +12,52 @@ version: 2.1
 orbs:
   win: circleci/windows@2.2.0
 
+_refs:
```

Comment:
Simple question, but I'd like to make sure my understanding is correct. Does this automatically kick off once we merge the release branch into master? Also, what will happen if we address minor doc changes by merging into master now?

Reply:
> Does this automatically kick off once we merge the release branch into master?

That is correct, the release process will now be to just merge the release branch into the master branch.

> what will happen if we address minor doc changes by merging into master now?

We should talk about those on a case by case basis, hopefully we won't have to do many of those but if we do we can add `[skip ci]` to the commit message which will prevent circleci from running the release job.